### PR TITLE
Bump Golang version to 1.19

### PR DESCRIPTION
**Pull request by ameukam**
Go 1.18 is [EOL](https://endoflife.date/go)
 
Signed-off-by: Arnaud Meukam <ameukam@gmail.com>

### CODE REVIEW

**Comment by aojea** [comment]
/lgtm
/hold
in case something breaks

**Comment by ameukam** [comment]
> link

@aojea 😅 

```sh
prow/gangway/client/google/google.go:23:2: SA1019: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
	"io/ioutil"
```

I'll send a commit in this PR to switch the new package.

File: .go-version [hunk 1]
```diff
@@ -1 +1 @@
-1.18
+1.19
```
Comment (BenTheElder) [inline .go-version:1]: we'll have to get on 1.20+ by august 🙃

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve
thanks!

**Pull request merged**